app/src/main/java/org/odk/odknotifications/SyncDataWithServices.java

```diff
@@ -0,0 +1,52 @@
+package org.odk.odknotifications;
+
+import android.app.ProgressDialog;
+import android.os.AsyncTask;
+
+import org.odk.odknotifications.Activities.MainActivity;
+import org.odk.odknotifications.DatabaseCommunicator.DBHandler;
+import org.odk.odknotifications.Model.Notification;
+
+import java.util.ArrayList;
+
+public class SyncDataWithServices extends AsyncTask<Void,Void,Void> {
```

Comment:
Avoid using AsyncTask now, as it is deprecated in latest Android 30 version, which I think we would be supporting next. https://developer.android.com/reference/android/os/AsyncTask
Try using executor or something else.

Reply:
Done.